api/src/test/java/org/motechproject/nms/api/osgi/KilkariControllerBundleIT.java

```diff
@@ -38,10 +39,8 @@
 
 import javax.inject.Inject;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
I think I have that Intellij setting correct on one of my clients but not the other. And I don't think we run PMD on our test code, which is why this was missed. Will fix in my next checkin.